nuxeo-management-rest-api/src/test/java/org/nuxeo/rest/management/TestDistributionObject.java

```diff
@@ -68,9 +72,25 @@ public void testDistribution() throws IOException {
         try (CloseableClientResponse response = get()) {
             assertEquals(200, response.getStatus());
             JsonNode node = mapper.readTree(response.getEntityInputStream());
-            JsonNode foo = node.get("foo");
-            assertNotNull(foo);
-            assertEquals("bar", foo.asText());
+            JsonNode key = node.get(Environment.PRODUCT_NAME);
+            assertNotNull(key);
+            key = node.get(Environment.PRODUCT_VERSION);
+            assertNotNull(key);
+            key = node.get(Environment.DISTRIBUTION_NAME);
+            assertNotNull(key);
+            key = node.get(Environment.DISTRIBUTION_VERSION);
+            assertNotNull(key);
+            key = node.get(Environment.DISTRIBUTION_SERVER);
+            assertNotNull(key);
+            key = node.get(Environment.DISTRIBUTION_DATE);
+            assertNotNull(key);
+            ArrayNode bundles = (ArrayNode) node.get(Environment.BUNDLES);
+            assertNotNull(bundles);
+            assertTrue(bundles instanceof ArrayNode);
```

Comment:
Useless or should be done before the cast

Reply:
You might want to have a look at the [JsonAssert](https://github.com/nuxeo/nuxeo/blob/master/nuxeo-core/nuxeo-core-io/src/test/java/org/nuxeo/ecm/core/io/marshallers/json/JsonAssert.java) helper, that has interesting methods to validate JSON such as `contains()` for arrays.

Reply:
if I check for childrenContains("name"), I get all the names. but then it throws an execption because there is more than 1 result.
If I check anything else, i keep getting null pointers.
I don't have the source code of this helper it seems.

Reply:
It's here: https://github.com/nuxeo/nuxeo/blob/10.10/nuxeo-core/nuxeo-core-io/src/test/java/org/nuxeo/ecm/core/io/marshallers/json/JsonAssert.java, you should have it in your workspace.

But it's not really suitable here, it is useful when manipulating a JSON string or an entity object, there you already have a `JsonNode` from the read stream.
Then IMO you can just check that:
- the array is not empty
- its first element has a non null "name"
- its first element has a non null "version"

Reply:
Done that,
Please disregard the other comment about JsonAssert

Reply:
actually one assert does the 3 checks:
`JsonAssert.on(flat).get(0).get("name").notNull();`

Only works if flat is an array, has a name property and that prop is not null